src/test/java/teammates/test/cases/datatransfer/AdminEmailAttributesTest.java

```diff
@@ -0,0 +1,187 @@
+package teammates.test.cases.datatransfer;
+
+import java.util.Arrays;
+import java.util.Calendar;
+import java.util.Date;
+import java.util.List;
+
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import com.google.appengine.api.datastore.Text;
+
+import teammates.common.datatransfer.attributes.AdminEmailAttributes;
+import teammates.common.util.Const;
+import teammates.common.util.FieldValidator;
+import teammates.common.util.StringHelper;
+import teammates.common.util.TimeHelper;
+import teammates.storage.entity.AdminEmail;
+import teammates.test.driver.StringHelperExtension;
+
+/**
+ * SUT: {@link AdminEmailAttributes}.
+ */
+public class AdminEmailAttributesTest extends BaseAttributesTest {
+
+    private FieldValidator fieldValidator;
+    private AdminEmailAttributes adminEmailAttributes;
+    private List<String> addressReceiverListString;
+    private List<String> groupReceiverListFileKey;
+    private String subject;
+    private Text content;
+    private Date date;
+
+    @BeforeClass
+    public void classSetup() {
+        addressReceiverListString = Arrays.asList("example1@test.com", "example2@test.com");
+        groupReceiverListFileKey = Arrays.asList("listfilekey", "listfilekey");
+        subject = "subject of email";
+        content = new Text("valid email content");
+        date = new Date();
+        fieldValidator = new FieldValidator();
```

Comment:
Why not simply assign some/all of the above in the variable definitions themselves?